Kernel/Arch/riscv64/PCI/Initializer.cpp

```diff
@@ -114,14 +109,16 @@ void initialize()
 
             switch (controller_compatibility) {
             case ControllerCompatible::ECAM: {
-                // FIXME: Make this use a nice helper function
+                auto reg = node.reg();
+
+                if (reg.size() != 1) {
+                    // FIXME: Some devices have a `reg-map` property which we may be able to use
```

Comment:
You probably mean `reg-names` here.
`pci-host-ecam-generic` doesn't use the `reg-names` property, you are probably thinking about the VF2 devicetree here, right?
The linux dt binding says the following:
```yaml
  reg:
    description:
      The Configuration Space base address and size, as accessed from the parent
      bus. The base address corresponds to the first bus in the "bus-range"
      property. If no "bus-range" is specified, this will be bus 0 (the
      default). Some host controllers have a 2nd non-compliant address range,
      so 2 entries are allowed.
    minItems: 1
    maxItems: 2
```
So I think you should always be able to use `reg[0]`.